collections/_posts/2023-01-30-Lipschitz_Anomaly.md

```diff
@@ -0,0 +1,80 @@
+
+# Experimental evaluation
+The authors propose two experiments that I will describe bellow.
+
+### 1) Training set contamination
+
+The authors propose to evaluate their model performance's with a classic easy anomaly detection task on MNIST : assuming one digit is the normal class and the others are anomaly (e.g. $$P_n$$ is a set of "3" and $$P_a$$ are "1" "4", etc.).
+
+They then add training set contamination (e.g. "5" in the training set when the normal class is "3") from 0% to 10%, compare to other SOTA methods and demonstrate the superiority of their method LAD (Table 1 with contamination fraction on top).
+
+They also show superiority of their method when combined to a Convolutional Auto-Encoder (LAD + CAE).
+
+
+![](/collections/images/lipschitz_anomaly/fig_results.jpg) 
+
+An impressive result is that the corruption process ($$P_n$$ to $$\hat{P_a}$$) used here on MNIST is patch shuffling (with 4x4 patches), which is very very far from the true anomaly distribution $$P_a$$. (e.g. an image of a "3" shuffled will look very different from an image of a "4").
+While $$\hat{P_a}$$ is very different from $$P_a$$, the results are competitive.
+
+### 2) Bias towards interpolated points
+
+The authors pointed out that the inductive bias from an Auto-Encoder reconstruction error was that the anomalies will be far from the mean of the training data in the latent space. They propose to investigate this bias in an experiment.
+
+#### Black image in MNIST
+They first show every anomaly detection method the "all black image". The all black image is very abnormal because it is not a digit, but very easy to reconstruct for an Auto-Encoder because it has learn to reconstruct black part of images. Result of the anomaly score for each method are shown in table 1 (represented by the ranking of the image, from 0 to 1 with 1 being ranked first).  
+Reconstructions based methods are CAE, DCAE, RCAE, AND, ALOCC and AnoGAN, SVM based methods are OCSVM and DSVDD and other machine learning methods IF and LOF.
+
+#### Mean image in CIFAR-10
+	
+When training on CIFAR-10 (normal class is one of the class and anomalies are other classes, like in MNIST), they find out that the performances vary greatly among classes (table	2).  
+They explain this by looking at the mean image for each class : in class such as birds, the samples are usually close to the mean image, whereas in cars for instance, images can be very far from the mean image (when with white background for example). This is also shown in table 2 where they give the rank of the mean image.
+
+To showcase this even more, they display the top 100 "most normal" images in the test set when trained on cars in figure 4. We see that the LAD has images that are far from the mean (i.e. with clear background), and that it mistakes trucks for cars, but not animals for cars as the reconstruction based method.
+
+![](/collections/images/lipschitz_anomaly/fig_bias.jpg) 
+
```

Comment:
Many images appear on the collections/images/lipschitz_anomaly folder but they are not displayed in the review